worldender/worldender/models/choice.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import BaseModel, Field
+from typing import List
+
+class Choice(BaseModel):
+    """
+    Choice is a possible choice to make in response to an Event
+    This choice will have an impact on the world and how the story progresses
+    It could lead to the world ending sooner, or later
+    """
+    choice: str = Field(
+        description="The choice to make in response to the event"
+    )
+    consequence: str = Field(
```

Comment:
so where will this consequence be shown in the UI? 

Maybe this could be, like, an interstitial, so while the LLM is figuring out the next event, we could display this consequence. Or, also, this should be passed to the LLM as part of the choice?

Reply:
Yes this should be passed to the LLM to form a chain of thought, I like the display idea too.